examples/angular/app.component.spec.ts

```diff
@@ -15,14 +15,14 @@ describe('AppComponent', () => {
   let fixture: ComponentFixture<AppComponent>;
   let app: AppComponent;
 
-  beforeEach(waitForAsync(() => {
-    TestBed.configureTestingModule({
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
```

Comment:
Import of `beforeEach` was missing. After adding it, the test is failing. This change fixed the test. Reference: https://angular.io/guide/testing-components-scenarios#consolidated-setup